boards/riscv32/m2gl025_miv/doc/m2g1025_miv.rst

```diff
@@ -0,0 +1,69 @@
+.. _m2gl025-miv:
+
+Microsemi M2GL025 Mi-V
+######################
+
+Overview
+********
+
+The Microsemi M2GL025 board is an IGLOO2 FPGA based development board.
+The Mi-V RISC-V soft CPU can be deployed on the MGL025 board.
+More information can be found on
+`Microsemi's website <https://www.microsemi.com/product-directory/embedded-processing/4406-cpus>`_.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``m2gl025_miv`` board configuration can be built as usual
+(see :ref:`build_an_application`).
+In order to build the application for ``m2gl025_miv``, set the ``BOARD`` variable
+to ``m2gl025_miv``.
+
+.. code-block:: bash
+
+   export BOARD="m2gl025_miv"
+
+Flashing
+========
+
+In order to upload the application to the device, you'll need OpenOCD and GDB
+with RISC-V support.
+You can get them as a part of SoftConsole SDK.
+Download and installation instructions can be found on
+`Microsemi's SoftConsole website
+<https://www.microsemi.com/product-directory/design-tools/4879-softconsole>`_.
+
+With the necessary tools installed, you can connect to the board using OpenOCD.
+To establish an OpenOCD connection run:
+
+.. code-block:: bash
+
+   sudo LD_LIBRARY_PATH=<softconsole_path>/openocd/bin \
+   <softconsole install path>/openocd/bin/openocd  --file \
```

Comment:
Should all three of these placeholders be ``<softconsole_path>``?  Not sure why the second one is different.

Reply:
Thanks for catching that. It is now fixed.